scripts/rmc

```diff
@@ -68,9 +70,11 @@ def main():
     if "--function" not in args.cbmc_args:
         args.cbmc_args.extend(["--function", args.function])
     
-    args.cbmc_args.append(str(RMC_C_LIB))
+    args.c_lib.append(str(RMC_C_LIB))
+    args.cbmc_args.extend(args.c_lib)
```

Comment:
why extend them here?

Reply:
We need to pass them to `cbmc` for it to know about the C files.

Reply:
My understanding was that the c files were in that goto file

Reply:
If not, lets add them there and not change the cbmc args